src/index.js

```diff
@@ -607,6 +607,95 @@ export default function (babel) {
     path.unshiftContainer("body", declarations);
   }
 
+  function generateForInIterator (path, type: "array" | "object") {
+    let idx = path.node.idx || path.scope.generateUidIdentifier("i");
```

Comment:
I just copied what was done from previous versions of this code - I'll update to `const`.